sopel/tools/__init__.py

```diff
@@ -31,16 +31,13 @@
 from ._events import events  # NOQA
 from . import time, web  # NOQA
 
-if sys.version_info.major >= 3:
-    raw_input = input
-    unicode = str
-    iteritems = dict.items
-    itervalues = dict.values
-    iterkeys = dict.keys
-else:
-    iteritems = dict.iteritems
-    itervalues = dict.itervalues
-    iterkeys = dict.iterkeys
+
+# Kept for backward compatibility
+# TODO: consider removing that
```

Comment:
For the record, I would be quite happy to deprecate these, as they have no place in Sopel's `tools` module.

It could be done in a separate PR though, and if you agree just lmk and I'll add it to a task list somewhere.

Reply:
Yup, agree, later.

Reply:
Tossed on my to-deprecate list. 👍